app/views/coronavirus_landing_page/components/landing_page/_stay-home-logo.html.erb

```diff
@@ -0,0 +1,48 @@
+<% alt_text = "" %>
+
+<svg class="covid__logos" width="300" height="45" viewBox="0 0 300 45" fill="none" xmlns="http://www.w3.org/2000/svg" aria-labelledby="covid-logo__title">
+  <title id="covid-logo__title"><%= alt_text %></title>
```

Comment:
Do we need this if the alt_text is empty?

Reply:
We can probably remove this

Reply:
Done